### PR TITLE
archive HadoopOverview -> data/hadoop-overview.md

### PR DESCRIPTION
**Pull request by edquist**
https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/HadoopOverview

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Add migration header to TWiki document

### CODE REVIEW

File: docs/data/hadoop-overview.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<span class="twiki-macro DOC_STATUS_TABLE"></span>
```
Comment (bbockelm) [inline docs/data/hadoop-overview.md]: Remove twiki cruft.

File: docs/data/hadoop-overview.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+--------
+
+-   [HadoopUnderstanding](https://twiki.opensciencegrid.org/bin/view/Documentation/HadoopUnderstanding): This document explains the various components for Hadoop and the recommended specifications of size and deployment.
+
+Installation
+------------
+
+-   [Install Guide](install-hadoop-2-0-0)
+    -   Change link <https://twiki.grid.iu.edu/bin/view/Accounting/ProbeInstallation>
+    -   <https://twiki.grid.iu.edu/bin/view/Trash/ReleaseDocumentationGratiaSiteCollector> revisit this link
```
Comment (bbockelm) [inline docs/data/hadoop-overview.md]: ???

File: docs/data/hadoop-overview.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+**Upgrading:** Please note that there are two upgrades that could be needed. To upgrade if you are using 0.19 from the Caltech repository, follow the first guide to upgrade to 0.20 then use the second to convert to the newer OSG repository.
+
+-   [HadoopUpgrade](https://twiki.opensciencegrid.org/bin/view/Storage/HadoopUpgrade) : Upgrade guide to upgrade from 0.19 to 0.20, both using the Caltech repo.
+-   [UpgradeHadoop](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/UpgradeHadoop): This guide covers upgrading from 0.20 Caltech repository to 0.20 using OSG repository.
```
Comment (bbockelm) [inline docs/data/hadoop-overview.md]: Remove both, no site is on 0.20 or 0.19.

File: docs/data/hadoop-overview.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+---------
+
+-   [HadoopOperations](https://twiki.opensciencegrid.org/bin/view/Storage/HadoopOperations): Information on how to run hadoop client commands, fscks, and other manual tips.
+-   [HadoopRecovery](hadoop-recovery): Notes and tips on how to recover corrupted or lost files.
+-   [HadoopDebug](troubleshooting-hadoop): Troubleshooting tips. Always a work in progress. Help us add solutions!
+
+Additional Links
+----------------
+
+-   [HadoopXrootd](https://twiki.opensciencegrid.org/bin/view/Storage/HadoopXrootd) : Special considerations when installing Xrootd on top of Hadoop
```
Comment (bbockelm) [inline docs/data/hadoop-overview.md]: Reasonable to have this page, but it seems to predate modern Xrootd configs and plugins.

@zvada - what's the best place for this?

**Comment by brianhlin** [comment]
Yea, I think we can probably slurp most of this stuff into the install doc but by my count, we'll need ~5 hadoop docs:

- install https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/InstallHadoop200SE
- understanding https://twiki.opensciencegrid.org/bin/view/Documentation/HadoopUnderstanding
- debug https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/HadoopDebug
- recovery https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/HadoopRecovery
- operations https://twiki.opensciencegrid.org/bin/view/Storage/HadoopOperations
- xrootd? https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/HadoopOverview

File: docs/data/hadoop-overview.md (hunk 4 above)
Comment (brianhlin) [reply docs/data/hadoop-overview.md]: pinging @zvada

**Comment by edquist** [comment]
Roger.

File: docs/data/hadoop-overview.md (hunk 4 above)
Comment (zvada) [reply docs/data/hadoop-overview.md]: I think for HadoopXrootd we should put it under Xrootd general doc as a subpage. I need to migrate XRootD main page first to see where it fits best. I plan on massive doc migrations which I assigned to me next days. Then I can see right fit of HadoopXrootd. Is that acceptable?

Comment (brianhlin) [reply docs/data/hadoop-overview.md]: Marian, hold off on doc migration for now, I hope to have a better solution so you don't have to spend much time on the migration process

**Pull request merged**